rigex-front-end/package.json

```diff
@@ -2,7 +2,9 @@
   "name": "rigex-front-end",
   "version": "0.1.0",
   "private": true,
+  "prettier": "@jasonephraim/prettier-config",
```

Comment:
I could be wrong but I don't think anything that has your @ should be in a folder that everyone gets. That seems like a problem

Reply:
fixed